_pages/en/web-services-access.md

```diff
@@ -1,63 +1,57 @@
-Even though TBS policy recommends open access to these tools by default, ESDC's rationale for blocking them is primarily to prevent an accidental leak of protected information by employees.
+### Background
 
-A proposed mitigation item to reduce this risk is to augment the Data Loss Prevention (DLP) capability with a tool that enforces classification of documents and emails.
-However, as the evaluated options to date are costly, implementation is not being considered for the short term.
+The upcoming [Policy on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603) replaces the [Policy on Acceptable Network and Device Use (PANDU)](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=27122) but keeps the intent to "provide open access to Internet tools and services to support public servants in their work duties, enhance collaboration and networking with their peers".
+See the [Examples of Acceptable Network and Device Use](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32605).
 
-The following Web tools can already be accessed on the ESDC network and are being used by employees (See [Appendix B](#appendix-b---web-tools-already-accessible)).
+### Current Status
 
-### Considerations
+ESDC is currently blocking access to some popular and useful collaboration Web tools and SaaS like Google Docs, Google Hangouts and OneDrive (see [Appendix A](#appendix-a---web-tools-currently-blocked)).
+The rationale for blocking them is primarily to prevent an accidental leak of protected information by employees.
 
-Departments are leveraging Web tools to collaborate and engage with citizens and public administrations both in Canada and across the globe.
-For example, the latest Open Government Plan and Service and Digital Target Architecture from TBS were shared as drafts for consultation and comments on Google Docs.
-The GC Digital Academy and Free Agents, as well as other groups, frequently use Google Forms for registration.
+A proposed mitigation item to reduce this risk is to augment the Data Loss Prevention (DLP) capability with a tool that enforces classification of documents and emails.
+However, implementation is not being considered for the short term.
 
-In order to improve ESDC's ability to leverage current Web tools, a balanced approach is proposed for granting access to additional Web tools (See [Appendix A](#appendix-a---web-tools-to-grant-access)) only for IITB employees in the short term, as a step toward broader access in ESDC.
-As IITB employees generally don't deal with protected information and are well informed and trained on the management of information, the risk would be minimal.
-In addition, the benefits should bring great value such as enhancing collaboration and efficiency, increasing job satisfaction and helping attract and retain IT talent.
+A firewall upgrade is underway and configuration for categories and sites blocked or unblocked will be documented and reviewed.
```

Comment:
Why is this important? Is it because with the previous firewall version we did not have the ability to have categories of sites? And instead if was a long list of individual sites?
Need clarity as to why firewall upgrade is required for this.